docker/Dockerfile.build.aarch64-linux

```diff
@@ -16,7 +18,23 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     pkg-config \
     libtool \
     libtool-bin \
-    && rm -rf /var/lib/apt/lists/*
+    python3-distutils \
+    zip \
+    python3.6 python3.6-dev \
```

Comment:
How does it work that you install Python here and later compile it and install again? Do we need both install commands, does the second one only produce headers?

Reply:
We are not compiling, we just configure and make inclinstall which install the headers.

Reply:
understood